components/statistic/Statistic.tsx

```diff
@@ -36,11 +38,27 @@ const Statistic: React.FC<StatisticProps & ConfigConsumerProps> = props => {
     valueRender,
     prefix,
     suffix,
+    loading,
     direction,
     onMouseEnter,
     onMouseLeave,
   } = props;
-  const valueNode = <StatisticNumber {...props} value={value} />;
+  let skeletonProps: SkeletonProps = { loading: false };
+  if (typeof loading === 'boolean') {
```

Comment:
default config active Skeleton is good enough. No need to pass additional prop to inner skeleton in case it may block future refactor.

Reply:
done.